charts/newrelic-infrastructure/values.yaml

```diff
@@ -90,24 +86,11 @@ ksm:
   # Advanced users only. Setting this to `false` is not supported and will break the New Relic experience.
   enabled: true
   annotations: {}
-  tolerations:
-    - operator: "Exists"
-      effect: "NoSchedule"
-    - operator: "Exists"
-      effect: "NoExecute"
```

Comment:
This change is definitely not user friendly. Users will think the default is `[]` (no tolerations), while it's not. I'd keep the tolerations here if possible.

Reply:
Inb4 "but the global then..."
I strongly believe users using such advanced features should manually go here and set `tolerations` to `null` if they want their global to be honored.

Reply:
I got affinity and tolerations back to the values and changed tests accordingly.

`tolerations` roll back solved in [#6dc65d5](https://github.com/newrelic/nri-kubernetes/pull/383/commits/6dc65d59737a550ce3b82aca406fb2891dd848b0)
`affinity` roll back solved in [#583772b](https://github.com/newrelic/nri-kubernetes/pull/383/commits/583772be24ee6debf2db48717295fd8b68ec9970)

Reply:
This one in particular solved in [#cc3aee0](https://github.com/newrelic/nri-kubernetes/pull/383/commits/cc3aee07d53624c7ea805f80fbf4e7dfe2c32aad)